internal/graphql/model/operator_helper.go

```diff
@@ -25,22 +25,23 @@ const (
 	MeshsyncSubject = "meshery.meshsync.core"
 	BrokerQueue     = "meshery"
 
-	operatorYaml = "https://raw.githubusercontent.com/layer5io/meshery-operator/master/config/manifests/default.yaml"
-	brokerYaml   = "https://raw.githubusercontent.com/layer5io/meshery-operator/master/config/samples/meshery_v1alpha1_broker.yaml"
+	// operatorYaml = "https://raw.githubusercontent.com/layer5io/meshery-operator/master/config/manifests/default.yaml"
+	// brokerYaml   = "https://raw.githubusercontent.com/layer5io/meshery-operator/master/config/samples/meshery_v1alpha1_broker.yaml"
 )
 
-func Initialize(client *mesherykube.Client, delete bool) error {
+func Initialize(client *mesherykube.Client, delete bool, adapterTracker models.AdaptersTrackerInterface) error {
 	// installOperator
-	err := applyYaml(client, delete, operatorYaml)
+	err := installUsingHelm(client, delete, adapterTracker)
 	if err != nil {
 		return err
 	}
 
+	// not needed
 	// installBroker
-	err = applyYaml(client, delete, brokerYaml)
-	if err != nil {
-		return err
-	}
+	// err = applyYaml(client, delete, brokerYaml)
```

Comment:
we can remove the commented code, if there is no reason to use yamls going forward.